content/docs/0.14.x/bridge/manage_sequences/index.md

```diff
@@ -65,13 +65,20 @@ Fields:
 link="./assets/trigger-05-trigger-evaluation-timeframe"
 caption="Sequences">}}
 
-#### **Start / end date**
-Please mind, that an evaluation is performed in the past. That means that the start date has to be **after** the end date.
+Special considerations:
+* If you set a timeframe but no start date, the evaluation will be performed to the past for the given timeframe. This is the only case where evaluations will be performed to the past.
```

Comment:
If you set a timeframe but no start date, the evaluation is performed on the previous timeframe span.  This is the only case where evaluations are performed on data from the past.